core/controlplane/config/templates/cloud-config-controller

```diff
@@ -1344,6 +1351,51 @@ write_files:
         - kind: ServiceAccount
           name: heapster
           namespace: kube-system
+  # Heapster's pod_nanny monitors the heapster deployment & its pod(s), and scales
+  # the resources of the deployment if necessary.
+  - path: /srv/kubernetes/rbac/roles/pod-nanny.yaml
+    content: |
+        apiVersion: rbac.authorization.k8s.io/v1beta1
+        kind: Role
+        metadata:
+          name: system:pod-nanny
+          namespace: kube-system
+          labels:
+            kubernetes.io/cluster-service: "true"
+            addonmanager.kubernetes.io/mode: Reconcile
+        rules:
+        - apiGroups:
+          - ""
+          resources:
+          - pods
+          verbs:
+          - get
+        - apiGroups:
+          - "extensions"
+          resources:
+          - deployments
+          verbs:
+          - get
+          - update
```

Comment:
A blank line here would be nice to make it easier to see the boundaries of each file.